pokemongo_bot/cell_workers/move_to_fort.py

```diff
@@ -140,22 +150,26 @@ def _get_nearest_fort_on_lure_way(self, forts):
 
     def get_nearest_fort(self):
         forts = self.bot.get_forts(order_by_distance=True)
-
         # Remove stops that are still on timeout
-        forts = filter(lambda x: x["id"] not in self.bot.fort_timeouts, forts)
+        forts = filter(
+            lambda x: x["id"] not in self.bot.fort_timeouts or (
+                x.get('active_fort_modifier', False) and self.wait_at_fort
+            ),
+            forts
+        )
```

Comment:
Nope, cause we dont need to care about forts_on timeout here. We standing and waiting for lures and spin fort with lure module on timeout(check spin_fort.py update L186)